### PR TITLE
amam/create signup

### PR DESCRIPTION
**Pull request by mustofa-binary**

### CODE REVIEW

**Review by suthesank — DISMISSED** [review]

File: src/common/utility.js [hunk 1]
```diff
@@ -58,6 +58,21 @@ const sanitize = input => input.replace(/[.*+?^${}()|[\]\\]/g, '')
 
 const sentenceCase = input => input.charAt(0).toUpperCase() + input.slice(1)
 
+const snackbar_options = {
+    position: 'bottom-right',
+    style: {
+        backgroundColor: '#ffc71b',
+        color: '#0e0e0e',
```
Comment (ghost) [inline src/common/utility.js:65]: Is it better to use naming colors from `themes`?

Comment (mustofa-binary) [reply src/common/utility.js:65]: themes color can only injected on the children inside provider

File: src/pages/home/_hero.js [hunk 2]
```diff
@@ -41,6 +48,56 @@ export const Hero = () => {
         return errors
     }
 
+    const getVerifyEmailRequest = email => {
+        const utm_data = TrafficSource.getData()
+        const affiliate_token = Cookies.getJSON('affiliate_tracking')
```
Comment (ghost) [inline src/pages/home/_hero.js]: Why did you use Cookies instead of CookieStorage to get the cookie?

Comment (mustofa-binary) [reply src/pages/home/_hero.js]: affiliate_tracking used CookieStorage at TrafficSource to write it, but, to get it, i want to extract the whole cookie value as an object

**Review by ghost — APPROVED** [review]

**Pull request merged**